docs/uptime/settings.asciidoc

```diff
@@ -0,0 +1,27 @@
+[role="xpack"]
+[[uptime-settings]]
+
+== Settings
+
+[role="screenshot"]
+image::uptime/images/settings.png[Filter bar]
+
+The Uptime settings page lets you change which Heartbeat indices are displayed
+by the uptime app. Users must have the 'all' permission to modify items on this page.
+Uptime settings apply to the current space only. Use different settings in different
+spaces to segment different uptime use cases and domains.
+
+As an example, imagine your organization one team for internal IT services, and another
+for public services. Each team operates independently and is only responsible for their
+own services. In this scenario you might setup separate Heartbeat instances for each team,
+writing out to index patterns named `it-heartbeat-\*`, and `external-heartbeat-\*`. You would
+create separate roles and users for each in Elasticsearch, each with access to their own spaces,
+named `it` and `external` respectively. Within each space you would navigate to the settings page
+and set the correct index pattern to match only the indices that space is allowed to access. 
```

Comment:
This example was a bit tough for me to read and comprehend. Here's an alternative that I came up with, but I'm not sure I like it any better:

> As an example, imagine your organization has one team for internal IT services, and another for public services. Each team operates independently and is only responsible for its own services. In this scenario, you might set up separate Heartbeat instances for each team, writing to different index patterns, named `it-heartbeat-*`, and `external-heartbeat-*`. In Elasticsearch, you'd create separate roles and users, named `it` and `external`, each exclusively with access to their own spaces. Finally, within each space, you would navigate to the settings page and set an index pattern only to match the indices that the space is allowed to access.

Maybe @dedemorton can take a look. She's pretty good at word herding.